### PR TITLE
release notes july 20

### PR DESCRIPTION
**Pull request by Slashek**

### CODE REVIEW

**Review by diana-lakatos — APPROVED** [review]
Reviewed and edited, thank you!

File: app/views/pages/api-reference/liquid/flow-control.liquid [hunk 1]
```diff
@@ -5,6 +5,10 @@ metadata:
   description: Liquid Tags - Flow Control
   toc: true
 ---
+aaa
```
Comment (pavelloz) [inline app/views/pages/api-reference/liquid/flow-control.liquid:8]: What is this @diana-lakatos  ?

File: app/views/pages/api-reference/liquid/variables.liquid [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 converter: markdown
+enable_profiler: true
```
Comment (pavelloz) [inline app/views/pages/api-reference/liquid/variables.liquid:3]: This shouldnt go to production @diana-lakatos

File: app/views/pages/api-reference/liquid/flow-control.liquid (hunk 1 above)
Comment (diana-lakatos) [reply app/views/pages/api-reference/liquid/flow-control.liquid:8]: @Slashek Please answer, I only did the review for the release note copy and the navigation. Thank you!

File: app/views/pages/api-reference/liquid/variables.liquid (hunk 2 above)
Comment (diana-lakatos) [reply app/views/pages/api-reference/liquid/variables.liquid:3]: @Slashek Please advise, thank you.

File: app/views/pages/api-reference/liquid/flow-control.liquid (hunk 1 above)
Comment (diana-lakatos) [reply app/views/pages/api-reference/liquid/flow-control.liquid:8]: I didn’t do these changes, I only reviewed and edited the release note and the release note navigation page. I fixed this. Thank you!

File: app/views/pages/api-reference/liquid/variables.liquid (hunk 2 above)
Comment (diana-lakatos) [reply app/views/pages/api-reference/liquid/variables.liquid:3]: Fixed, thank you!

**Pull request merged**